kolibri/plugins/coach/api.py

```diff
@@ -180,6 +183,8 @@ def get_queryset(self):
                 )
             except (LearnerProgressNotification.DoesNotExist):
                 return []
+            except DatabaseError:
+                repair_sqlite_db(connections["notifications_db"])
```

Comment:
do you think we need to return [] here?

Reply:
yes, I have added

Reply:
oh I guess you were trying to return [] after `repair_sqlite_db`?

Reply:
:man_facepalming: yes, of course. I should not code at some hours in the day...